### ISSUE TITLE
Building docs locally fails with weird  error.

### ISSUE THREAD

**Issue by dwf** [bartvm/blocks#506]
Specifically,

```
  File "conf.py", line 313, in fix_apply
RuntimeError: dictionary changed size during iteration
```

Did this change recently?

**Comment by bartvm** [comment]
Don't think so, but Sphinx as well as napoleon recently released new versions, and everything's a mess :/ The docs don't render correctly anymore (you'll see that every brick has the docstring of `Application` rendered), etc. Have been meaning to figure this out, but haven't had time.

**Comment by dwf** [comment]
It gets worse, #507.

**Comment by dwf** [comment]
So, apparently iterating over `__dict__.items()` when deleting attributes is a non-starter in Python 3, at least. Fixed by #508.
